### ISSUE TITLE
DB sources and destinations: Catch bad private key as config error

### ISSUE THREAD

**Issue by edgao** [airbytehq/airbyte#18518]
e.g. https://github.com/airbytehq/oncall/issues/893

```
SshTunnel.java
Function: getPrivateKeyPair
Type: java.io.StreamCorruptedException

Missing end marker ([END RSA PRIVATE KEY]) after line #1
```

In this case, the user just didn't have paste their entire private key correctly. This shouldn't get reported to sentry.

The relevant code is (IIRC) shared across sources and destinations, so tagging for both teams.

**Comment by akashkulk** [comment]
This should be fixed in the next version 1.0.20

**Comment by bleonard** [comment]
https://github.com/airbytehq/oncall/issues/720
